errand_matcher/templates/errand_matcher/health-and-safety.html

```diff
@@ -19,14 +19,14 @@
 </head>
 
 <body>
-	<div class="top-nav-bar">
-		<ul class="nav-list">
-			<li class="nav-list-item button home-link" id="livelyhood-list-item">LivelyHood</li>
-			<li class="nav-list-item button health-and-safety-link">Health and safety protocol</li>
-			<li class="nav-list-item button faq-link">FAQs</li>
-			<li class="nav-list-item button about-link">About</li>
-			<li class="nav-list-item button about-link">Contact</li>
-		</ul>
+	<div class="topnav" id="topnav">
+		<!-- TODO: Check all links are correct. -->
+		<a class="topnav-item button home-link" id="home-item">LivelyHood</a>
+		<a class="topnav-item button health-and-safety-link" id="collapsible-item">Health and safety protocol</a>
```

Comment:
Each of these id's should be unique, i.e. id="collapsible-item-1", "collapsible-item-2"

Reply:
Updated, meant to use class for these. Thanks!